### DISCUSSION TITLE
warning with tailwind.config.js

### DISCUSSION THREAD

**Discussion by zsh28** [tailwindlabs/tailwindcss#14141] [Help]
```
/** @type {import('tailwindcss').Config} */
export default {
  content: ["./src/**/*.{js,ts,jsx,tsx}"],
  theme: {
    fontFamily: {
      jost: ['Jost', 'sans-serif'],
      inconsolata: ['Inconsolata', 'monospace'],
    },
    screens: {
      '2xl': { 'max': '1200px' },
    },
  },
  plugins: [],
  mode: 'jit',
}
```
warning in console:
warn - The `min-*` and `max-*` variants are not supported with a `screens` configuration containing objects.

not sure what I have done wrong: I followed this docs https://tailwindcss.com/docs/screens#max-width-breakpoints

**Comment by wongjn** [comment]
I don't believe you are doing anything inherently wrong with regards to the `screens` configuration. However – like the message states – the `min-*` and `max-*` variants are not supported with your `screens` configuration containing objects. See https://github.com/tailwindlabs/tailwindcss/pull/9558#user-content-restrictions for details.
